source/tyr/tyr/binarisation.py

```diff
@@ -811,3 +811,45 @@ def cosmogony2mimir(self, autocomplete_instance, filename, job_id, dataset_uid):
         job.state = 'failed'
         models.db.session.commit()
         raise
+
+
+@celery.task(bind=True)
+def poi2mimir(self, instance_name, input, job_id=None, dataset_uid=None):
```

Comment:
What is `dataset_uid` used for ?

Reply:
dataset_uid was not used, good catch... and it was supposed to be used to collect metrics.